### PR TITLE
docs: document cfg, WSL installation, Source Sans

### LINKED ISSUE #203: document state of WSL support
Just a quick note that I tested texml on a recent windows 11 with the standard ubuntu WSL. 

Installation went fine and processing seems to mostly work. 

However, I'm running into svg-creation / xelatex problems again. I'll update the card as I move along.

- SVG creation: xelatex cannot find STIX Two
  - I downloaded the otf files and copied them as per https://gist.github.com/rajeshkumaravel/2795c341f4adf9daffb1791dd5bd3004
  - `sudo cp STIX* /usr/local/share/fonts/ && sudo fc-cache -fv`
  -  sourcesanspro not found
     -  workaround: adding something for `math_sf_font` in the cfg file

### PR DESCRIPTION
**Pull request by pkra**
Resolves #203

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,12 +142,17 @@ Starting with an Ubuntu 18.04.5 LTS installation:
...
+
+The above instructions should work in an Ubuntu-based [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
 
 ## Testing
 
-If you install the STIX Two fonts somewhere where fontconfig can find
+If you install the STIX Two fonts and Source Sans Pro somewhere where fontconfig can find
 them, you might also be able to compile test/graphics.tex.  (If you
 want to be able to use stix2.sty (or stix.sty), you'll need the Ubuntu
 fonts-stix package (I think).)
```
Comment (pkra) [inline README.md:158]: I was wondering about this line.

Comment (davidmjones) [reply README.md:158]: > I was wondering about this last sentence.

@pkra Sorry, I'm confused.  What is the question?  You addition makes sense.  Is there something else?

Comment (pkra) [reply README.md:158]: @davidmjones sorry, I was unclear.

I'm wondering if we want to update (or remove) the following:

>  (If you
want to be able to use stix2.sty (or stix.sty), you'll need the Ubuntu
fonts-stix package (I think).)

But now I had time to take another look. From what I can tell, `fonts-stix` (debian/ubuntu)  installs STIX 1.1. 

I think `tex-live-fonts-extra` includes STIX Two and SourceSans (but comes in at an extra 2GB so perhaps a warning and alternate path should be documented).

Comment (pkra) [reply README.md]: I've tweaked things some more following our last discussion

**Pull request merged**